### PR TITLE
id clarifications

### PR DESCRIPTION
**Pull request by TanyaStere42**
location IDs have complicated IDs. clarifying how to make them and how users should deal with them

### CODE REVIEW

File: docs/data/bulk-import-beta.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ In the header of an Excel (.xlsx) file, create a column for each field you want
     If an encounter is associated with a [Sighting](../introduction/sighting.md), include the needed information for the Sighting on at least one line of an associated Encounter.
 * If using any of the Occurrence fields for Sightings, ensure that each Encounter is linked with a common value in the Occurrence.occurrenceID column.
 * Verify the following fields match exactly with what exists in the system:
-    * **Encounter.locationID** - Must match the location as shown in the Location ID menu of the Report an Encounter page.
+    * **Encounter.locationID** - Must match the location as shown in the LocationID.json of your branch.
```
Comment (goddesswarship) [inline docs/data/bulk-import-beta.md]: This doesn't help the average Wildbook user, whose only experience with WIldbook will be on the product site. Can we combine both? "Must match the location as shown in the LocationID.json of your branch or the Location ID menu of the Report an Encounter page."

File: docs/data/bulk-import-beta.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ The most commonly-used fields are listed in the table below.
 | MarkedIndividual.individualID | V_WString | Lion045 | PrimaryKey. While Wildbook assigns unique user IDs to MarkedIndividuals, this field becomes the default display name for the MarkedIndividual. <br>Duplicate of Encounter.individualID. <br>We recommend using this field instead of Encounter.individualID. |
 | Encounter.individualID | V_WString | Lion045 | Duplicate of MarkedIndividual.individualID. |
 | Encounter.verbatimLocality | V_WString | Saw this about five minutes into our tour near mile marker 5, somewhere in Tanzania. | Descriptive string to reference location. No constraints on what is entered, but is not used for analysis. |
-| Encounter.locationID | V_WString | Zone 5, Botswana | Hierarchical list of study sites. Allows for consistency and controlled granularity of location input. **Must match the location as shown in the Location ID menu of the Report an Encounter page.** |
+| Encounter.locationID | V_WString | Zone 5, Botswana | Hierarchical list of study sites. Allows for consistency and controlled granularity of location input. **Must match the location as shown in the LocationID.json of your branch.** |
```
Comment (goddesswarship) [inline docs/data/bulk-import-beta.md]: Ditto above.

**Review by goddesswarship — COMMENTED** [review]
Minor changes suggested so that instructions remain friendly to the non-tech savvy.

File: docs/data/bulk-import-beta.md (hunk 1 above)
Comment (TanyaStere42) [reply docs/data/bulk-import-beta.md]: The issue is one of those isn't true and we're setting up our users to input the wrong information if we tell them to use the report an encounter page information

Comment (goddesswarship) [reply docs/data/bulk-import-beta.md]: How so? How is the average Wildbook user supposed to infer that we're referring to code and not something on their Wildbook page?

**Review by goddesswarship — APPROVED** [review]
🦖

**Pull request merged**